zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/actuator/ClusterActuator.java

```diff
@@ -70,6 +71,8 @@ static ClusterActuator of(final String endpoint) {
         .encoder(new JacksonEncoder(List.of(new Jdk8Module(), new JavaTimeModule())))
         .decoder(new JacksonDecoder(List.of(new Jdk8Module(), new JavaTimeModule())))
         .retryer(Retryer.NEVER_RETRY)
+        // The default http client do not support http PATCH
+        .client(new feign.httpclient.ApacheHttpClient())
```

Comment:
:exploding_head: I bet that was a fun one to figure out

Reply:
ya.. took the whole day :smile: